Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-ruby "3.1.3"
+ruby "3.1.4"
```

Comment:
Why are you changing your ruby version?

Reply:
It's because one of our team members has Ruby version 3.1.4, while the rest of us have version 3.1.3. However, soon we will try to ensure that everyone has version 3.1.3.